documentation/source/BenchmarkingYoloNAS.md

```diff
@@ -0,0 +1,45 @@
+# Benchmarking YoloNAS
+## Introduction:
+
+YoloNAS is a leading object detection architecture that combines accuracy and efficiency. Using quantization-aware training (QAT), YoloNAS models can be optimized for resource-constrained devices.
+However, to fully tap into its potential, it is crucial to know how to export the quantized model to the INT8 TensorRT (TRT) engine.
+In this blog, we emphasize the significance of this step and provide a concise guide to efficiently exporting a quantized YoloNAS model to the INT8 TRT engine.
+Doing so teaches us how to properly benchmark YoloNAS and understand its full potential.
+
+## Step 1: Export YoloNAS to ONNX
+
+The first step is to export our YoloNAS model to ONNX correctly. Two actions must be taken before we export our model to onnx:
+1. We need to replace our layers with "fake quantized" ones - this happens when we perform post-training quantization or quantization-aware training with SG.
+So nothing to worry about if you performed PTQ/QAT with SG and hold your newly exported ONNX checkpoint.
+2. We must call model.prep_model_for_conversion - this is essential as YoloNAS incorporates QARepVGG blocks. Without this call, the RepVGG branches will not be fused, and our model's speed will decrease significantly!
+Again, nothing to worry about if you have quantized your model with PTQ/QAT with SG, as this is done under the hood before exporting the ONNX checkpoints.
```

Comment:
I'd also emphasize that the inference time of the model (PyTorch) with the Q/DQ layers is slower, and they should not benchmark it unless it is already compiled to an engine

Reply:
also, since the PyTorch version is with the QARepVGG blocks unmerged, it is even slower.

Reply:
Changed in 5d03e150a1c9aad2f21864d3a535a2aac8c48a44